code/modules/paperwork/pen.dm

```diff
@@ -119,18 +122,28 @@
 	if(!M.can_inject(user, TRUE))
 		return
 	var/transfered = 0
+	var/contained = list()
+	for(var/R in reagents.reagent_list)
+		var/datum/reagent/reagent = R
+		contained += "[reagent.volume]u [reagent]"
 	if(reagents.total_volume && M.reagents)
-		transfered = reagents.trans_to(M, 50)
+		transfered = reagents.trans_to(M, transfer_amount)
 	to_chat(user, "<span class='warning'>You sneakily stab [M] with the pen.</span>")
-	add_attack_logs(user, M, "Stabbed with (sleepy) [src]. [transfered]u of reagents transfered.")
+	add_attack_logs(user, M, "Stabbed with (sleepy) [src]. [transfered]u of reagents transfered from pen containing [english_list(contained)].")
 	return TRUE
 
 
 /obj/item/pen/sleepy/Initialize(mapload)
 	. = ..()
-	create_reagents(100)
-	reagents.add_reagent("ketamine", 100)
+	create_reagents(reagent_amount)
+	for(var/key in starting_reagents)
+		reagents.add_reagent(key, starting_reagents[key])
 
+/obj/item/pen/sleepy/vv_edit_var(var_name, var_value)
+	. = ..()
+	switch(var_name)
+		if("reagent_amount")
+			reagents.maximum_volume = var_value
```

Comment:
Yeah I aint having a var exist just as a VV callback.

People can edit the thing.